src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -122,7 +122,7 @@ class TxQ_test : public TestSuite
     }
 
 public:
-    void run()
+    void testQueue()
```

Comment:
There is movement on the committee in this area.  I have seen proposals that would allow this to work without the `move`.  I _think_ they haven't been voted in yet; checking on that.  We are probably looking at vendors gaining field experience in this area to support such a proposal.